pyspark/python_package/python_submit_yarn_client.sh.template

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This template is to show how to submit python job with dependencies on spark yarn clusters.
+
+BigDL_HOME=/root/jwang/BigDL
+SPARK_HOME=/opt/work/spark-1.6.3-bin-hadoop2.6
+PYTHON_API_ZIP_PATH=${BigDL_HOME}/dist/lib/bigdl-0.1.0-SNAPSHOT-python-api.zip
+BigDL_JAR_PATH=${BigDL_HOME}/dist/lib/bigdl-0.1.0-SNAPSHOT-jar-with-dependencies.jar
+PYTHONPATH=${PYTHON_API_ZIP_PATH}:$PYTHONPATH
+
+http_proxy=http://...
```

Comment:
this is an example, if user does not need to set proxy, he can remove it. I change the filename as example instead of template